Product/SoapUI_Test/FileUtils/src/main/java/nhinc/FileUtils.java

```diff
@@ -71,7 +74,7 @@
      */
     public static String readFile(String fileName, Logger log) {
         try {
-            log.info("read file: " + fileName);
+            log.info(MessageFormat.format("read file: {0}", fileName));
```

Comment:
No need to use MessageFormat.format. You can simple use log.info("read file {0}", fileName);  This applies to all comments below

Reply:
currently the version: org.apache.log4j.Logger used in the FileUtils; don't have the capability of formatting string